### PR TITLE
Additional annotations needed static pods

### PR DESCRIPTION
**Pull request by Warren3225**
You need an additional annotation called ad.datadoghq.com/endpoints.resolve if you're running endpoint checks on a service that sits in front of static pods. I'm not sure how to hyperlink the example to the documentation so I'm referencing it below and in the edit I'm making. Thanks!

https://docs.datadoghq.com/containers/kubernetes/control_plane/?tab=helm#api-server-2

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/containers/cluster_agent/endpointschecks.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ The Agents connect to the Cluster Agent every ten seconds and retrieve the check
 
 This feature is supported on Kubernetes for Agent v6.12.0+ and Cluster Agent v1.3.0+. Starting with v1.4.0, the Cluster Agent converts every Endpoints Check of a non-pod-backed endpoint into a regular Cluster Check. Enable the [Cluster Check][4] feature alongside Endpoints Checks to take advantage of this functionality.
 
+NOTE: If the pods behind your service are static pods, you'll need another annotation ad.datadoghq.com/endpoints.resolve. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See an example of this annotation with the kube API server here - https://docs.datadoghq.com/containers/kubernetes/control_plane/?tab=helm#api-server-2
```
Comment (maycmlee) [inline content/en/containers/cluster_agent/endpointschecks.md]:
```suggestion
**Note:** If the pods behind your service are static pods, add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][] of using the annotation with the kube API server. https://docs.datadoghq.com/containers/kubernetes/control_plane/?tab=helm#api-server-2
```

File: content/en/containers/cluster_agent/endpointschecks.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ The Agents connect to the Cluster Agent every ten seconds and retrieve the check
 
 This feature is supported on Kubernetes for Agent v6.12.0+ and Cluster Agent v1.3.0+. Starting with v1.4.0, the Cluster Agent converts every Endpoints Check of a non-pod-backed endpoint into a regular Cluster Check. Enable the [Cluster Check][4] feature alongside Endpoints Checks to take advantage of this functionality.
 
+NOTE: If the pods behind your service are static pods, you'll need another annotation ad.datadoghq.com/endpoints.resolve. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [an example][5] of this annotation with the kube API server.
```
Comment (maycmlee) [inline content/en/containers/cluster_agent/endpointschecks.md:27]:
```suggestion
**Note:** If the pods behind your service are static pods, you need to add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][5] of the annotation with the Kubernetes API server.
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

File: content/en/containers/cluster_agent/endpointschecks.md (hunk 2 above)
Comment (Warren3225) [reply content/en/containers/cluster_agent/endpointschecks.md:27]: LGTM

Comment (maycmlee) [inline content/en/containers/cluster_agent/endpointschecks.md:27]:
```suggestion
**Note:** If the pods behind your service are static pods, you need to add the annotation `ad.datadoghq.com/endpoints.resolve`. The Datadog Cluster Agent schedules the checks as endpoint checks and dispatches them to Cluster Check Runners. See [this example][5] of using the annotation with the Kubernetes API server.
```

**Pull request merged**